star-wars.Rmd

```diff
@@ -5,16 +5,31 @@ output: github_document
 
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(knitr)
 ```
 
 ## Most accomplished
 
 Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the fewest number of starships):
```

Comment:
If they are the most accomplished they should have flown the **most** starships.

```suggestion
Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the fewest number of starships):
```

Reply:
oh they are the same! sorry nothing